user.js

```diff
@@ -82,10 +82,10 @@ user_pref("browser.aboutConfig.showWarning", false);
 user_pref("_user.js.parrot", "0100 syntax error: the parrot's dead!");
 /* 0102: set START page [SETUP-CHROME]
  * [SETTING] General > Thunderbird Start Page ***/
-user_pref("mailnews.start_page.enabled", false);
+// user_pref("mailnews.start_page.enabled", false);
```

Comment:
(will be already addressed in #52)

Reply:
a bit confused here, this is not modified in my branch, and I would not modify it here. I leave it without the commenting lines